src/Microsoft.Extensions.Configuration.AzureAppConfiguration/LogHelper.cs

```diff
@@ -74,5 +74,15 @@ public static string BuildPushNotificationUnregisteredEndpointMessage(string res
         {
             return $"{LoggingConstants.PushNotificationUnregisteredEndpoint} '{resourceUri}'.";
         }
+
+        public static string BuildFailoverToDifferentEndpointMessage(string originalEndpoint, string currentEndpoint)
+        {
```

Comment:
For the message it looks like it's currently

"Failed to get configuration settings from endpoint '{old endpoint}'. Trying to read from endpoint '{new endpoint}'."

I'm thinking to make the fall back clear:

"Failed to get configuration settings from endpoint '{old endpoint}'. Falling back to endpoint '{new endpoint}'."

Reply:
That's a good idea, can we use "failover" instead of "fallback" though? That's the term we use in public docs too.

"Failed to get configuration settings from endpoint '{old endpoint}'. Initiating automatic failover to endpoint '{new endpoint}'."

Reply:
Yes. "Failing over to endpoint ..."